packages/aml-checklist/src/components/ui/modal/address-information/form-schema/validation-schema.ts

```diff
@@ -0,0 +1,21 @@
+import * as Yup from 'yup'
+import errorMessages from '@/constants/error-messages'
+import { isValidUploadForm } from '@/utils/validates'
+
+/* istanbul ignore next */
+const validationSchema = Yup.object().shape({
+  metadata: Yup.object().shape({
```

Comment:
Please avoid hard-coded key name in validation schema, import and use them  from form-field instead